### PR TITLE
Update wearcompose to v1.3.0-alpha04

### PR DESCRIPTION
**Pull request by yschimke**
From https://github.com/google/horologist/pull/1611

### CODE REVIEW

File: auth/composables/src/test/snapshots/images/com.google.android.horologist.auth.composables.screens_SelectAccountScreenTest_selectAccountScreen.png
Comment (yschimke) [inline auth/composables/src/test/snapshots/images/com.google.android.horologist.auth.composables.screens_SelectAccountScreenTest_selectAccountScreen.png:1]: @luizgrp I wasn't expecting this.

File: media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components.actions_ShowPlaylistChipTest_givenNOArtwork_thenDoesNOTDisplayArtwork.png
Comment (yschimke) [inline media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components.actions_ShowPlaylistChipTest_givenNOArtwork_thenDoesNOTDisplayArtwork.png:1]: @luizgrp nor this

File: auth/composables/src/test/snapshots/images/com.google.android.horologist.auth.composables.screens_SelectAccountScreenTest_selectAccountScreen.png
Comment (yschimke) [reply auth/composables/src/test/snapshots/images/com.google.android.horologist.auth.composables.screens_SelectAccountScreenTest_selectAccountScreen.png:1]: I might look at putting the now in android updates PR auto updates in place, and we can tighten the tolerances.

Comment (luizgrp) [reply auth/composables/src/test/snapshots/images/com.google.android.horologist.auth.composables.screens_SelectAccountScreenTest_selectAccountScreen.png:1]: Confirming the above, I've just ran the tests from this class in record mode, in main, and they were updated.

Comment (luizgrp) [reply auth/composables/src/test/snapshots/images/com.google.android.horologist.auth.composables.screens_SelectAccountScreenTest_selectAccountScreen.png:1]: We should balance the tolerance so we don't miss important changes.

File: media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components.actions_ShowPlaylistChipTest_givenNOArtwork_thenDoesNOTDisplayArtwork.png
Comment (luizgrp) [reply media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components.actions_ShowPlaylistChipTest_givenNOArtwork_thenDoesNOTDisplayArtwork.png:1]: Also ran this in main and it got updated, so not a side effect from this upgrade. It sounds right that the text is centred in this Chip when secondary label is not present.

**Pull request merged**